sklearn/tree/tree.py

```diff
@@ -1628,6 +1628,19 @@ class ExtraTreeRegressor(DecisionTreeRegressor):
 
     .. [1] P. Geurts, D. Ernst., and L. Wehenkel, "Extremely randomized trees",
            Machine Learning, 63(1), 3-42, 2006.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import load_boston
+    >>> from sklearn.model_selection import cross_val_score
+    >>> from sklearn.ensemble import BaggingRegressor
+    >>> from sklearn.tree import ExtraTreeRegressor
+    >>> X, y = load_boston(return_X_y=True)
+    >>> extra_tree = ExtraTreeRegressor()  # doctest: +ELLIPSIS
+    >>> reg = BaggingRegressor(extra_tree, n_estimators=10, random_state=0)
+    >>> cross_val_score(reg, X, y, cv=5)
```

Comment:
Using `cross_val_score` for a example in the docstring seems a little too computational intensive. I'll be okay with `fit` on a train set and score on a test set.